### PR TITLE
[feature] show dynamic attributes in API Resources index table

### PR DESCRIPTION
**Pull request by donrestarone**
Addresses: https://github.com/restarone/violet_rails/issues/871

**Demo:**

https://user-images.githubusercontent.com/25191509/179969416-1d807c22-9c82-46d1-91b9-bc27bbdbce2c.mp4

### CODE REVIEW

**Comment by donrestarone** [comment]
@alis-khadka this is on its way to Heroku!

File: test/controllers/admin/comfy/api_namespaces_controller_test.rb [hunk 1]
```diff
@@ -467,4 +467,23 @@ class Comfy::Admin::ApiNamespacesControllerTest < ActionDispatch::IntegrationTes
     assert_match success_message, request.flash[:notice]
     assert_equal api_namespace_hash['api_namespace']['name'], ApiNamespace.last.name
   end
+
+  test "should allow #show and the dynamic properties should be shown in the api-resources tables" do
+    sign_in(@user)
+    properties = {"attr_1"=>true, "attr_2"=>true, "attr_3"=>true, "attr_4"=>true}
+    @api_namespace.update(properties: properties)
+    @api_namespace.api_form.update(properties: {
```
Comment (donrestarone) [inline test/controllers/admin/comfy/api_namespaces_controller_test.rb:475]: is `@api_namespace.api_form.update(properties:` updating the API form necessary @alis-khadka ?

**Comment by donrestarone** [comment]
@alis-khadka lets make the table responsive so it doesn't break the viewport (see how its going outside the user's field of view on the right handside). 

Screenshot from 2022-07-20 20-10-48

File: test/controllers/admin/comfy/api_namespaces_controller_test.rb (hunk 1 above)
Comment (alis-khadka) [reply test/controllers/admin/comfy/api_namespaces_controller_test.rb]: Just realized that there is a after_save callback for updating the api_form. I have refactored the test case.

**Comment by alis-khadka** [comment]
@donrestarone 
I have made the table scrollable when the content is bigger than the viewport.

**Demo**

https://user-images.githubusercontent.com/25191509/180190218-50c86786-86c2-441e-aa6b-071b0121e48e.mp4

**Pull request merged**